external/easylogging++/easylogging++.cc

```diff
@@ -1015,8 +1015,9 @@ const std::string OS::getBashOutput(const char* command) {
   char hBuff[4096];
   if (fgets(hBuff, sizeof(hBuff), proc) != nullptr) {
     pclose(proc);
-    if (hBuff[strlen(hBuff) - 1] == '\n') {
```

Comment:
Can you help me understand the policies around external/ libraries? If we change this should a PR be sent to the original maintainer? Is it important that we keep these "pure"?

Reply:
A PR to the original maintainer is best, yes.